server/apps/main/templates/main/application.html

```diff
@@ -53,5 +52,8 @@
 
     <script src="{% static '/js/main.js' %}"></script>
 
+    <!-- bootstrap tab -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
```

Comment:
Thanks! This is very helpful. 
- took the bootstrap.min.js out and updated the CSS to 4.3.1 to match with the existing min.js. 
- added a "rejected" tab
- "In review" tab now collects public stories that are "in review" and "not reviewed". Because its slightly awkward to have a tab "In Review" and stories with "In Review" "Not Reviewed" in it, I changed the tab name to "Pending review"

Reply:
That looks great and works like a charm now!